variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+# List of variables related to power plants
+# Variables related to the CO2 emission by power plant
+CO2 Emmissions|Electricity|Biomass|Traditionnal:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Gas|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+# Variables related to the production cost of a power plant
```

Comment:
suggest to add this explanation to the `variables/README.md` or as a separate md readme file